api/src/org/apache/cloudstack/api/command/user/template/CreateTemplateCmd.java

```diff
@@ -215,33 +221,39 @@ public long getEntityOwnerId() {
         if (volumeId != null) {
             Volume volume = _entityMgr.findById(Volume.class, volumeId);
             if (volume != null) {
-                accountId = volume.getAccountId();
```

Comment:
accountId is unused. Can we initialize it with CallContext.current().getCallingAccount() and use it (which saves some function calls)?